docs/get-started/enterprise-and-community-edition.md

```diff
@@ -99,9 +99,9 @@ Meanwhile, users can use any deep models to execute the live streams on the fly.
 
 ## Community
 
-There is a developer community. You can join the google group, and ask or answer questions: [https://community.antmedia.io/](https://community.antmedia.io/)
+There is a user community available. You can ask or answer questions by joining the community at [GitHub Discussions](https://github.com/orgs/ant-media/discussions)
 
-You can also ask your questions at StackOverflow with the ```ant-media-server``` tag.
+You can also ask your questions in [Discussions Q&A](https://github.com/orgs/ant-media/discussions/categories/q-a)
 
```

Comment:
I think this can be a one-liner, something like this. 

Join the Ant Media Server community over at GitHub Discussions and ask or answer questions in the Discussions Q&A.